src/script/view_model/content/PreferencesAccountViewModel.ts

```diff
@@ -119,14 +147,14 @@ z.viewModel.content.PreferencesAccountViewModel = class PreferencesAccountViewMo
 
     this.isTeam = this.teamRepository.isTeam;
     this.team = this.teamRepository.team;
-    this.teamName = ko.pureComputed(() => t('preferencesAccountTeam', this.teamRepository.teamName()));
+    this.teamName = ko.pureComputed(() => t('preferencesAccountTeam', this.teamRepository.getTeamName()()));
 
     this.optionPrivacy = ko.observable();
     this.optionPrivacy.subscribe(privacyPreference => {
       this.propertiesRepository.savePreference(PROPERTIES_TYPE.PRIVACY, privacyPreference);
     });
 
-    this.optionReadReceipts = this.propertiesRepository.receiptMode;
+    this.optionReadReceipts = this.propertiesRepository.getReceiptMode();
```

Comment:
Why using a method call here instead of using the property directly?

Reply:
receiptMode is a private member in PropertiesRepository

Reply:
You can make it a read-only public member.

Reply:
https://github.com/wireapp/wire-webapp/pull/9073/commits/20aa1d2943e5dcd65dd596d8c9200ea557d1f82d